taskfile/read/taskfile.go

```diff
@@ -152,6 +152,12 @@ func Taskfile(readerNode *ReaderNode) (*taskfile.Taskfile, error) {
 		if err = taskfile.Merge(t, includedTaskfile, includedTask.Internal, namespace); err != nil {
 			return err
 		}
+
+		if includedTaskfile.Tasks["default"] != nil && t.Tasks[namespace] == nil {
+			t.Tasks[namespace] = &taskfile.Task{}
+			*t.Tasks[namespace] = *(includedTaskfile.Tasks["default"])
+		}
```

Comment:
Hey @pd93!

Now that we have a whole `alias` infra in the code, I think we should change how we plan to do this.

Instead of copying the whole task here, we should add an aliases to the task pointing to the alternative names. This way `task --list` will also behave as expected, not duplicating tasks and documenting the right aliases as well.

```yaml
version: '3'

includes:
  docs:
    aliases: [d]
    taskfile: ./docs
    dir: ./docs
```

```yaml
# docs/Taskfile.yml

version: '3'

tasks:
  default:
    desc: Something
    cmds:
      - ...
```

`task -l` should print something like this:

```
$ task -l
task: Available tasks for this project:
* docs:default:         Something                              (aliases: docs, d, d:default)
```

Reply:
This is a great idea. I've pushed a change to use aliases instead